img/u/uottawa.ca.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" viewBox="0 0 55.98 64.27"><defs><style>.cls-1{fill:#84776f}</style></defs><path d="M13.71 7.4H6.78v3.63H0v53.24h55.98V11.03H49.2V7.4h-6.93C39.32 2.64 34.3 0 27.99 0S16.66 2.64 13.71 7.4Zm.81 2.98h.89l.42-.78c2.97-5.47 8.25-6.62 12.16-6.62s9.18 1.15 12.16 6.62l.42.78h5.65v3.64H53v47.27H2.98V14.01h6.78v-3.64h4.76Z" class="cls-1"/><path d="M6.08 53.31h43.81v1.64H6.08v-1.64zM4.8 56.18h46.38v1.64H4.8v-1.64zM27.49 17.36 5.58 25.24l1.01 2.8s20.44-7.35 21.4-7.69c.96.35 21.4 7.69 21.4 7.69l1.01-2.8-22.41-8.06-.5.18ZM46.86 48.7l-.41-16.69c.13.07.27.11.43.11.51 0 .93-.41.93-.93 0-.25-.1-.48-.26-.65h.75V28.9H7.68v1.64h.74a.935.935 0 0 0 .67 1.58c.15 0 .29-.04.41-.1L9.1 48.7H7.56v2.73h40.85V48.7h-1.55Zm-29.27 0H12.9l-.41-16.68c.13.07.27.11.42.11.51 0 .93-.41.93-.93 0-.25-.1-.48-.26-.65h3.33a.935.935 0 0 0 .67 1.58c.15 0 .29-.04.42-.11l-.4 16.68Zm8.5 0h-4.7l-.41-16.68a.928.928 0 0 0 1.35-.83c0-.25-.1-.48-.26-.65h3.35a.935.935 0 0 0 .67 1.58c.15 0 .29-.04.41-.1l-.41 16.68Zm8.48 0h-4.68l-.41-16.67c.12.06.25.09.39.09.51 0 .93-.41.93-.93 0-.25-.1-.48-.26-.65h3.35a.935.935 0 0 0 .67 1.58c.15 0 .3-.04.42-.11l-.41 16.68Zm8.49 0h-4.68l-.41-16.68c.12.06.26.1.41.1.51 0 .93-.41.93-.93 0-.25-.1-.48-.26-.65h3.34a.935.935 0 0 0 .67 1.58c.15 0 .29-.04.41-.1l-.41 16.68Z" class="cls-1"/></svg>
```

Comment:
This one could probably be made smaller with the use of objects and object referencing but in order to save time I've just removed some redundant coordinates.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#84776f" viewBox="0 0 55.98 64.27"><path d="M13.71 7.4H6.78v3.63H0v53.24h55.98V11.03H49.2V7.4h-6.93C39.32 2.64 34.3 0 27.99 0S16.66 2.64 13.71 7.4Zm1.7 2.98c2.6-4.94 7.71-7.4 12.58-7.4s9.95 2.5 12.58 7.4h5.65v3.64H53v47.27H2.98V14.01h6.78v-3.64c1.89-.01 3.77.01 5.65.01z"/><path d="M6.08 53.31h43.81v1.64H6.08v-1.64zM4.8 56.18h46.38v1.64H4.8v-1.64zm22.69-38.82L5.58 25.24l1.01 2.8 21.4-7.69 21.4 7.69 1.01-2.8-22.41-8.06-.5.18ZM46.86 48.7l-.41-16.69a.93.93 0 0 0 1.36-.82c0-.25-.1-.48-.26-.65h.75V28.9H7.68v1.64h.74a.94.94 0 0 0 .67 1.58c.15 0 .29-.04.41-.1L9.1 48.7H7.56v2.73h40.85V48.7h-1.55Zm-29.27 0H12.9l-.41-16.68c.13.07.27.11.42.11.51 0 .93-.41.93-.93 0-.25-.1-.48-.26-.65h3.33a.94.94 0 0 0 .67 1.58c.15 0 .29-.04.42-.11l-.4 16.68Zm8.5 0h-4.7l-.41-16.68a.93.93 0 0 0 1.35-.83c0-.25-.1-.48-.26-.65h3.35a.94.94 0 0 0 .67 1.58c.15 0 .29-.04.41-.1l-.41 16.68Zm8.48 0h-4.68l-.41-16.67a.93.93 0 0 0 1.32-.84c0-.25-.1-.48-.26-.65h3.35a.94.94 0 0 0 .67 1.58c.15 0 .3-.04.42-.11l-.41 16.68Zm8.49 0h-4.68l-.41-16.68a.92.92 0 0 0 1.34-.83c0-.25-.1-.48-.26-.65h3.34a.94.94 0 0 0 .67 1.58c.15 0 .29-.04.41-.1l-.41 16.68Z"/></svg>
```